docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -595,6 +619,31 @@ Asset field is optional. When not set, it defaults to the asset user requested.
 
 :::
 
+##### Waiting For User Funds
+
+:::tip
+
+This step is optional. Most business don't use it: you can skip it and go to the [next one](#sending-stellar-transaction)
+
+:::
+
+After update in the previous chapter has been made, user will be notified that your service is ready to receive funds. In real world, transfer confirmation may take time. In such cases, transaction should be set to a new status, after funds have been received, but not confirmed yet.
```

Comment:
I would add a note that the business should only use this status if the off-chain payment method being used notifies them of a pending inbound payment.